packages/deprecation-crawler/tests/index.test.ts

```diff
@@ -0,0 +1,102 @@
+import * as fs from 'fs';
+import * as cp from 'child_process';
+import * as path from 'path';
+import * as rimraf from 'rimraf';
+
+const SANDBOX_PATH = path.join(__dirname, '..', 'sandbox');
+
+const testcases = fs
+  .readdirSync(SANDBOX_PATH)
+  .filter((name) => name !== 'deprecations') // output folder
+  .map((name) => path.join(SANDBOX_PATH, name))
+  .filter((name) => fs.lstatSync(name).isDirectory())
+  .map((testcase) => {
+    const input = path.join(testcase, 'input.ts');
+    const crawled = path.join(testcase, 'crawled.ts');
+    const output = path.join(testcase, 'output.ts');
+    return {
+      input,
+      crawled,
+      output,
+    };
+  });
+
+beforeAll(async () => {
+  testcases.forEach((testcase) => {
+    fs.copyFileSync(testcase.input, testcase.crawled);
+    rimraf(path.join(SANDBOX_PATH, 'deprecations'), (err) => {
+      if (err) {
+        console.error(err);
+      }
+    });
+  });
+});
+
+test('sandbox', async () => {
+  // BUG: format tag to be able to save the file, e.g. pr/foo
+  // const currentGitTag = await git([`branch --show-current`]);
+  const cliOutput = await exec('npm run crawl ' + 'master');
+
+  // verify output
+  expect(cliOutput).toMatch(/Looking for deprecations/i);
+  expect(cliOutput).toMatch(/Adding uuid to deprecations/i);
```

Comment:
Repository Unique Identifier :)
```suggestion
  expect(cliOutput).toMatch(/Adding ruid to deprecations/i);
```